mysql-plugin/src/e2e-test/resources/pluginParameters.properties

```diff
@@ -1,12 +1,36 @@
 driverName=mysql
 host=MYSQL_HOST
 username=MYSQL_USERNAME
+invalid.username=root1
 password=MYSQL_PASSWORD
-databaseName=sakila
+databaseName=testDatabase
 port=MYSQL_PORT
+numberofsplits=abc
+numberofsplitsgenerate=
+numbersplitsgenerate=2
+invalid.tablename=123#
```

Comment:
Separate the properties using comments. For example 
#AdvancedProperties
numberofsplits=abc

#Invalid Properties
property1=
property2=

Reply:
done